runtime/android/core/src/org/xwalk/core/XWalkCoreWrapper.java

```diff
@@ -13,194 +12,146 @@
 import android.util.Log;
 
 import java.lang.reflect.Method;
-import java.lang.reflect.InvocationTargetException;
 import java.security.MessageDigest;
 import java.security.NoSuchAlgorithmException;
+import java.util.LinkedList;
 
 import junit.framework.Assert;
 
-import org.xwalk.core.XWalkLibraryListener.LibraryStatus;
+/**
+ * <p>The appropriate call order is:
+ * handlePreInit() -> initXWalkLibrary() -> initXWalkEnvironment() -> handlePostInit() -> over</p>
+ */
+class XWalkCoreWrapper {
+    private static final String XWALK_APK_PACKAGE = "org.xwalk.core";
+    private static final String WRAPPER_PACKAGE = "org.xwalk.core";
+    private static final String BRIDGE_PACKAGE = "org.xwalk.core.internal";
+    private static final String TAG = "XWalkLib";
```

Comment:
This tag is shared by multiple classes to indicate that it's related to the library